react-function-components/src/index.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+
+function CustomButton() {
+  const myButton = <button> Click Me! </button>;
+  return myButton;
+}
+
+const container = document.querySelector('#root');
+const root = ReactDOM.createRoot(container);
+root.render(CustomButton());
```

Comment:
Here we don't want to call our component (in react we pretty much never call our own components. Instead, we use them to create a react element

```jsx
const element = <CustomButton />;
```